nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -943,31 +943,31 @@ private static com.sun.tools.javac.util.List<Type> convertTreesToTypes(
   /**
    * Substitutes the type arguments from a generic method invocation into the method's type.
    *
-   * @param methodInvocationTree the method invocation tree
+   * @param tree the method invocation tree
    * @param methodSymbol symbol for the invoked generic method
    * @param state the visitor state
    * @param config the NullAway config
    * @return the substituted method type for the generic method
    */
   private Type substituteTypeArgsInGenericMethodType(
-      MethodInvocationTree methodInvocationTree,
-      Symbol.MethodSymbol methodSymbol,
-      VisitorState state,
-      Config config) {
+      Tree tree, Symbol.MethodSymbol methodSymbol, VisitorState state, Config config) {
```

Comment:
We should write at least one test involving a constructor declaring its own type variables to make sure we cover this case a bit, e.g.:
```java
public class Holder {
  String s;
  public <U extends @Nullable Object> Holder(U value) {
    s = String.valueOf(value);
  }
  static Holder make() {
    return new <@Nullable String>Holder(null);
  }
}
```
If this doesn't pass NullAway we can at least do a follow-up task for fixing.

Reply:
Looks like this test is still not there?